components/github/sources/new-pull-request/new-pull-request.mjs

```diff
@@ -0,0 +1,163 @@
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+import github from "../../github.app.mjs";
+import commonWebhook from "../common/common-webhook.mjs";
+import constants from "../common/constants.mjs";
+
+const DOCS_LINK =
+  "https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request";
+
+export default {
+  key: "github-new-or-updated-pull-request",
```

Comment:
```suggestion
  key: "github-new-pull-request",
```

Reply:
Thanks for the heads up! [This is actually an existing component:](https://pipedream.com/apps/github/triggers/new-or-updated-pull-request) when `webhook-events` was created to supposedly replace a few existing sources, their code was removed from the repo, but they were not unpublished, so they are still available to users.

That is why I used the same key and set the version to 1.0.0, to update that existing component.

Reply:
Ah, ok. But shouldn't the filename be `new-or-updated-pull-request.mjs` so they match?

Reply:
Also, we'll definitely move some of the code to a shared file once we start building more components in this fashion. Thanks for the feedback!

Reply:
@michelle0927 yes, that makes sense - I'll rename it, thanks for noting that.